packages/preview/bookletic/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "bookletic"
+version = "0.1.0"
+compiler = "0.11.0"
+repository = ""
+entrypoint = "src/lib.typ"
+authors = [
+  "Brenden Harrell <https://github.com/harrellbm>"
+]
+license = "Apache-2.0"
+description = "Create beautiful booklets with ease."
+categories = [ "book", "layout", "flyer" ]
+keywords = [ "book", "booklet", "pamphlet", "bulletin" ]
+exclude = [ "/example/basic.png", "/example/fancy.png" ]
```

Comment:
I think you can exclude the whole `example` directory, as it will generally not be useful to have a copy of it in the local cache.